index.html

```diff
@@ -54,7 +54,170 @@ <h2 id="name_title">I’m Chima Glad to see you!</h2>
                     </ul>
                 </nav>
             </div>                        
-        </section>        
+        </section>      
+        <section id="works">
+            <div class="card">
```

Comment:
- [ ] I have really appreciated the way you wrote your HTML code, so clear and follows the requirement, but I have some issues to raise concerning the design

<img width="775" alt="3" src="https://user-images.githubusercontent.com/77843050/212474881-49ee38e2-2867-4914-a3c3-be0009941160.png">

- [ ]  Let start by the space between the headline and the first card, seeing the Figma image on the left, yours is smaller, please fix that up
- [ ] The background of your card section is darker, take a look at the image on the left and bring the necessary changes
- [ ] Your card width is to modify, consider adding it to match the original card design width

- [ ] Your card title's font-weight is to be touched a bit, make the necessary changes to have the same bold style as per the image on the left.